tests/invoke-tasks-test.sh

```diff
@@ -0,0 +1,56 @@
+KEYPAIR_NAME=ci.test-infra-definitions.test-key
+KEYPAIR_PATH=key-pair
+ENV=agent-qa
+
+echo "Creating ssh key pair"
+
+ssh-keygen -t ed25519 -C "Temporary key pair used for test-infra-definitions integration tests" -f key-pair
+
+echo "Import key pair on AWS"
+
+aws ec2 delete-key-pair --key-name ci.test-infra-definitions.test-key-$CI_PIPELINE_ID
```

Comment:
How is it done in the `datadog-agent` repository when running e2e tests?

Reply:
Just took what was done in `datadog-agent` to have things done the same way